packages/api/src/controllers/farmExpenseTypeController.js

```diff
@@ -25,9 +25,28 @@ const farmExpenseTypeController = {
       try {
         const data = req.body;
         data.expense_translation_key = data.expense_name;
-        const result = await baseController.postWithResponse(ExpenseTypeModel, data, req, { trx });
-        await trx.commit();
-        res.status(201).send(result);
+
+        // check if records exixts in DB
+        const record = await ExpenseTypeModel.query()
```

Comment:
suggestion: maybe we could add a `ExpenseTypeModel.existsInFarm(expense_name, farm_id)` method and just call it here so that 
1/ it’s more clear what we’re doing and we don’t need the comment 
2/ we don’t leak database schema detail in our http controller
3/ we can reuse it maybe